keyproof/progresslog_test.go

```diff
@@ -1,13 +1,15 @@
 package keyproof
 
+import "sync/atomic"
+
 type TestFollower struct {
-	count int
+	count int64
 }
 
 func (_ *TestFollower) StepStart(desc string, intermediates int) {}
 
 func (t *TestFollower) Tick() {
-	t.count++
+	atomic.AddInt64(&t.count, 1)
```

Comment:
There's a second TestFollower in representationproof_test.go which still uses count++. This one should use an atomic count, too.